docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -38,21 +38,17 @@ You can [play audio](#play_audio) and control functions for using audio resource
 
 The supported audio formats include MP3, AAC, WMA, M4A, 3GA, WAV, and AMR. The available formats depend on the target device.
 
-You can use multiple instances of the player to play several audio data streams concurrently. This means that your application can [play multiple uncompressed audio files](#play) at the same time.
+Software and hardware audio codec can be selected using the player API. You must check the default codec type by calling 'player_get_audio_codec_type()'.
```

Comment:
Instead of ', please use \` to use the code tag.

`player_get_audio_codec_type()`

Reply:
I pushed new commit. thank you.